src/ui-lib/sass/05-components/01-atoms/06-buttons/_08-toggle-menu.scss

```diff
@@ -1,31 +1,97 @@
 .grav-c-toggle-menu {
-  margin-top: 0;
-  border: $grav-st-thickness solid $grav-co-neutral-asphalt;
-  background-color: inherit;
-  color: $grav-co-bg;
 
-  &:hover {
-    background-color: inherit;
-    cursor: pointer;
-  }
+  $grav-sp-icon: 0.35rem;
+  $grav-tr-icon-hover: 0.1rem;
+  $grav-tr-icon-rotate: 135deg;
+  $grav-tr-icon-scale: 1.2;
+  $grav-size-icon: 33px;
 
+  display: flex;
+  height: $grav-size-icon;
+  margin-top: 0;
+  border-style: hidden;
+  
+  &--inverted,
+  .is-light & {
+    &__icon,
+    &__icon::before,
+    &__icon::after {
+      background-color: $grav-co-neutral-asphalt;
+    }
+     
+    &[aria-pressed='true']:hover,
+    &[aria-pressed='true']:active,
+    &[aria-pressed='true']:focus {
+      background-color: inherit;
+    }
+  }
+  
+  &,
```

Comment:
even if we're replicating the code, wouldn't it be better to have this property moved at the top together with the rest of the other properties? in this way you can see all at once what applies to what.